scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala

```diff
@@ -33,9 +33,6 @@ class ProcessMetaDataBuilder private[build](metaData: MetaData) {
   def slug(p: Option[String]) =
     new ProcessMetaDataBuilder(metaData.copy(typeSpecificData = RequestResponseMetaData(p)))
 
-  def subprocessVersions(subprocessVersions: Map[String, Long]) =
```

Comment:
I think that we need a migration guide entry that this API method is unavailable anymore

Reply:
Fixed.